daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -53,9 +54,17 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 | 1.0.0 or 1.0.1           |                   N/A |                    1.1.2 |
 |                          |                 1.1.2 |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.0 |
+|                          |                 1.3.0 |                    1.3.1 |
+|                          |                 1.3.1 |                    1.4.0 |
 | 1.1.0 to 1.1.2           |                   N/A |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.0 |
+|                          |                 1.3.0 |                    1.3.1 |
+|                          |                 1.3.1 |                    1.4.0 |
 | 1.2.0 to 1.2.2           |                   N/A |                    1.3.0 |
+|                          |                 1.3.0 |                    1.3.1 |
+|                          |                 1.3.1 |                    1.4.0 |
+| 1.3.0 or 1.3.1           |                   N/A |                    1.3.1 |
```

Comment:
Since specifically 1.3.0 and 1.3.1 are different in upgrade path, I think it makes sense not to group them together like the previous hotfixes in this table.

Reply:
done